R/pkg/R/mllib.R

```diff
@@ -694,8 +694,8 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, blockSize = 128, layers, solver = "l-bfgs", maxIter = 100,
+                   tol = 1E-6, stepSize = 0.03, seed = -763139545) {
```

Comment:
yeah, it is a problem.
now I consider a better way:

we give the `seed` parameter default value `null`
`MultilayerPerceptronClassifierWrapper.fit` add a `null` check for `seed` parameter,
if it is null, then do not call `MultilayerPerceptronClassifier.setSeed`
so it will automatically use the default seed.

how do you think about it ?